x/autopilot/keeper/fallback.go

```diff
@@ -0,0 +1,39 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Stride-Labs/stride/v16/x/autopilot/types"
+)
+
+// Stores a fallback address for an outbound transfer
+func (k Keeper) SetTransferFallbackAddress(ctx sdk.Context, channelId string, sequence uint64, address string) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.TransferFallbackAddressPrefix)
```

Comment:
at the end of the day, it doesn't make a difference, they both just cast a string to a byte array.

Fwiw, it's kinda the same as if you had a function like
```go
func KeyName(prefix []byte) string {
   return string(prefix)
}
```
and then called `KeyName(prefix)` instead of `string(prefix)`. 

Is a function really required? Probably not. 
Does it clarify things? Or does it make things more confusing if you don't know what it's doing off the top of your head? 🤷‍♂️ 

Curious what you think!

Reply:
As long as we never change from a byte array to another type then that makes sense. If it could possibly change down the line, then using the helper would prob be better. Small deal either way, let's keep whichever you prefer. Resolving.

Reply:
I believe it will always be a byte array unless the SDK makes a big unexpected change